code/modules/client/preferences_toggles.dm

```diff
@@ -104,15 +104,6 @@
 		src << "You will no longer prayer sounds."
 	feedback_add_details("admin_verb", "PSounds")
 
-/client/verb/togglePRs()
-	set name = "Show/Hide Pull Request Announcements"
-	set category = "Preferences"
-	set desc = "Toggles receiving a notification when new pull requests are created."
-	prefs.chat_toggles ^= CHAT_PULLR
-	prefs.save_preferences()
-	src << "You will [(prefs.chat_toggles & CHAT_PULLR) ? "now" : "no longer"] see new pull request announcements."
-	feedback_add_details("admin_verb","TPullR") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
-
```

Comment:
Before anyone asks, this is half removed because it took up a ton of space and you could do it without needing to go into the prefs tab, half removed because it doesn't even function on our server